Lib/venv/scripts/nt/activate.bat

```diff
@@ -1,11 +1,10 @@
 @echo off
 
-rem This file is UTF-8 encoded, so we need to update the current code page while executing it
-for /f "tokens=2 delims=:" %%a in ('"%SystemRoot%\System32\chcp.com"') do (
-    set "_OLD_CODEPAGE=%%a"
-)
+rem This file is UTF-8 encoded, so we need to update the current code page while executing it.
+for /f %%a in ('%~dp0python.exe -Ic "import ctypes; print(ctypes.windll.kernel32.GetConsoleOutputCP())"') do (set "_OLD_CODEPAGE=%%a")
+
 if defined _OLD_CODEPAGE (
-    "%SystemRoot%\System32\chcp.com" 65001 > nul
+    %~dp0python.exe -Ic "import ctypes; ctypes.windll.kernel32.SetConsoleOutputCP(65001)"
```

Comment:
On the tracker @zooba pointed that you're missing the quoting around `%~dp0python.exe`. I don't think the issue with the buildbot failures is related to spaces in the path, but remember to fix this problem as well.

Reply:
Good catch. I will do that. I will wait a few days until the design is cleared. My assumptions now are:
- we need a new PR starting of the up-to-date master
- we need to address the case that ctypes is not available with chcp.com
- we need quoting to support spaces in folder names
- we need to inject the python_exe name by a as a template variable such as __VENV_PYTHON_EXE_NAME__, assuming that this name is always ASCII
- we need a fallback in case __VENV_PYTHON_EXE_NAME__ is not available

Please correct me where you differ. I will make the PR as soon as you agree on the solution. Sorry to have broken the build.